dashboard/src/pages/SessionsPage.jsx

```
@@ -182,18 +195,36 @@ const SessionRow = React.memo(function SessionRow({ session, locale }) {
+            {isSubagent ? (
+              <span className="inline-flex items-center rounded-full bg-sky-50 px-2 py-0.5 text-[11px] font-medium text-sky-700 dark:bg-sky-500/10 dark:text-sky-300">
+                {copy("sessions.badge.subagent")}
+                {session.agent_role ? ` · ${session.agent_role}` : ""}
+              </span>
+            ) : null}
             {session.first_pass ? (
               <span className="inline-flex items-center rounded-full bg-emerald-50 px-2 py-0.5 text-[11px] font-medium text-emerald-700 dark:bg-emerald-500/10 dark:text-emerald-300">
                 {copy("sessions.badge.first_pass")}
               </span>
             ) : null}
+            {Boolean(childCount) ? (
```

ESLint

[error] Redundant Boolean call.

(no-extra-boolean-cast)